frontend/public/components/app.jsx

```diff
@@ -188,6 +188,8 @@ class App extends React.PureComponent {
             <LazyRoute path="/k8s/ns/:ns/secrets/:name/edit" exact kind="Secret" loader={() => import('./secrets/create-secret' /* webpackChunkName: "create-secret" */).then(m => m.EditSecret)} />
             <LazyRoute path="/k8s/ns/:ns/secrets/:name/edit-yaml" exact kind="Secret" loader={() => import('./create-yaml').then(m => m.EditYAMLPage)} />
 
+            <LazyRoute path="/k8s/ns/:ns/routes/new/form" exact kind="Route" loader={() => import('./routes' /* webpackChunkName: "create-route" */).then(m => m.CreateRoute)} />
```

Comment:
Does this mean we'll have both the project bar and the namespace selector in the form? That might be confusing.

Reply:
There is no namespace selector in the form.